modulus/datapipes/climate/dummy.py

```diff
@@ -0,0 +1,185 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023 - 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import time
+from typing import Any, Dict, List, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from torch.utils.data import DataLoader, Dataset
+
+
+class DummyWeatherDataLoader(DataLoader):
+    """
+    This custom DataLoader initializes the DummyWeatherDataset with given arguments.
+    """
+
+    def __init__(self, *args, **kwargs):
+        dataset = DummyWeatherDataset(*args, **kwargs)
+        super().__init__(
+            dataset=dataset,
+            batch_size=kwargs.get("batch_size", 1),
+            shuffle=kwargs.get("shuffle", False),
+            num_workers=kwargs.get("num_workers", 0),
+            pin_memory=kwargs.get("pin_memory", False),
+            drop_last=kwargs.get("drop_last", False),
+        )
+
+
+class DummyWeatherDataset(Dataset):
```

Comment:
Would prefer renaming to `SyntheticWeatherDataset` maybe

Reply:
Done